### PR TITLE
forces date use the default language for output

### PR DESCRIPTION
**Pull request by lrinQVQ**
This can be avoided https://github.com/lennylxx/ipv6-hosts/pull/160#discussion-diff-148743863R12

### CODE REVIEW

**Comment by lennylxx** [comment]
Cool! Thanks for the PR. 
This has already be tested on different machines, right?
And, could you please take care the indent issue in your change? we want our code clean and elegant.

File: merge_snippets.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ printf "\
 ## +-------------------------  >>d(' _ ')b<<  -------------------------+\r\n\
 ## |                                                                   |\r\n\
 ## |         Project: https://github.com/lennylxx/ipv6-hosts           |\r\n\
-## |         Update : `date    +"%a, %d %b %Y %T %z"`                  |\r\n\
+## |         Update : `LC_ALL="C" date    +"%a, %d %b %Y %T %z"`                  |\r\n\
```
Comment (lennylxx) [inline merge_snippets.sh]: I wonder LC_ALL="C" && date +"%a, %d %b %Y %T %z", will be better? With the AND op.

Comment (lennylxx) [reply merge_snippets.sh]: And, we don't need that much spaces before |\r\n\

**Comment by lrinQVQ** [comment]
Yes～I tested in Ubuntu 16.04/16.10/17.04/17.10/
Archlinux 
Centos
OS X
And 
Windows Bash

File: merge_snippets.sh (hunk 1 above)
Comment (lrinQVQ) [reply merge_snippets.sh]: Wow,
this is a good idea but LC_ALL is a variable.
We need add export for LC_ALL="C",make it effective in this shell.

Comment (lrinQVQ) [reply merge_snippets.sh]: maybe make the code longer XD

**Comment by lrinQVQ** [comment]
And I will processing indent issue,please wait~

File: merge_snippets.sh (hunk 1 above)
Comment (lennylxx) [reply merge_snippets.sh]: How about you add this before date command, say line 16. And make sure it only affect the current session.

Comment (lennylxx) [reply merge_snippets.sh]: And we can do it better, backup the current setting, once the script is finishing, restore the setting.

Comment (lrinQVQ) [reply merge_snippets.sh]: Sure～we can export first then unset this variable

**Pull request merged**